libs/pages/application/ui/src/lib/components/container/container.tsx

```diff
@@ -91,6 +93,47 @@ export function Container(props: ContainerProps) {
     </>
   )
 
+  const tabsItems = [
+    {
+      icon: <Icon name={IconEnum.CHECKCIRCLE} width="14" />,
+      name: 'Overview',
+      active: window.location.pathname === APPLICATION_URL(organizationId, projectId, environmentId, applicationId),
+      link: APPLICATION_URL(organizationId, projectId, environmentId, applicationId),
+    },
+    {
+      icon: <Icon name={IconEnum.CHECKCIRCLE} width="14" />,
+      name: 'Deployments',
+      active:
```

Comment:
Not a good practice to use `window.location.pathname` you must use the react-router
- https://surajsharma.net/blog/current-url-in-react